app/src/main/java/skills/future/planer/ui/day/views/matrix/MatrixFragment.java

```diff
@@ -55,12 +64,58 @@ public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
         recyclerViews.add(binding.notUrgentImportantRecycler);
         recyclerViews.add(binding.notUrgentNotImportantRecycler);
 
+        // background constrains
+        backgroundConstrains.add(binding.constrainUrgentImportant);
+        backgroundConstrains.add(binding.constrainUrgentNotImportant);
+        backgroundConstrains.add(binding.constrainNotUrgentImportant);
+        backgroundConstrains.add(binding.constrainNotUrgentNotImportant);
+
         setUpAdapters();
         setUpModels();
+        setUpBackground();
 
         return root;
     }
 
+    private void setUpBackground()
+    {
+        int colors[] = {Colors.getColorFromPreferences("urgentImportant", getContext()), 0x00ffffff, 0x00ffffff};
```

Comment:
Nie dało by się tego potwora krócej zapisać?

Reply:
nie wiem czemu nie wszystkie linijki zaznaczyło

Reply:
To do poprawnienia jest, ale chcieliście tego pulla na już